packages/common/src/context/comments/commentsContext.tsx

```diff
@@ -42,17 +42,17 @@ type CommentSectionContextType = {
   artistId: ID
   isEntityOwner: boolean
   commentCount: number
-  playTrack: () => void
   commentSectionLoading: boolean
   comments: Comment[]
   currentSort: TrackCommentsSortMethodEnum
   isLoadingMorePages: boolean
   hasMorePages: boolean
+  isMutating: boolean
+  setIsMutating: (isMutating: boolean) => void
+  playTrack: () => void
   reset: (hard?: boolean) => void
   setCurrentSort: (sort: TrackCommentsSortMethodEnum) => void
   loadMorePages: () => void
-  handleLoadMoreReplies: (commentId: string) => void
-  handleMuteEntityNotifications: () => void
```

Comment:
Another unrelated change - I noticed that we had these unimplemented methods in here leftover from before we decided to move the methods out to separate hooks